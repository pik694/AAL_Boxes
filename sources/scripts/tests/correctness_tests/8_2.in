8
74 40 11
84 44 48
16 33 94
70 84 78
88 50 35
78 94 16
20 73 79
94 65 47

