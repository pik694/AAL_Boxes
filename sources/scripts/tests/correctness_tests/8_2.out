0 6 3 2 5 4 1 7 
1392062->1066754
